Adding some new FAQs (#3385)
m3db/m3: site/content/faqs/faqs.md

- **How can I see the cardinality of my metrics?**
Currently, the best way is to go to the [M3DB Node Details Dashboard](https://grafana.com/grafana/dashboards/8126) and look at the `Ticking` panel. However, this is not entirely accurate because of the way data is stored in M3DB -- time series are stored inside time-based blocks that you configure. In actuality, the `Ticking` graph shows you how many unique series there are for the most recent block that has persisted. In the future, we plan to introduce easier ways to determine the number of unique time series. 

- **How can I check the health of my M3DB node or cluster?**

> Each M3DB node has an endpoint called Health, which you can query via HTTP: `curl localhost:9002/health

Reviewer: ```suggestion
Each M3DB node has a health endpoint, which you can query via HTTP: `curl localhost:9002/health
```
